<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioRepoDownloader</class>
 <widget class="QDialog" name="AudioRepoDownloader">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Base Updater</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutRepo">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Download to Album:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditDownloadLocation">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>You have to set album location...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonBrowse">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelSubtitle">
         <property name="text">
          <string>Select and click 'Download &amp; Add to Album' button.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="refreshPushButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton { padding: 2px; }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="repoSelectTree">
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Title</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxKeepDownload">
       <property name="title">
        <string>Keep Downloaded Files</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="comboBoxRepoList"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hButtonsLayout">
       <item>
        <widget class="QLabel" name="labelDownloadStatus">
         <property name="text">
          <string>Download Progress:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="downloadProgressBar">
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDownload">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Download &amp;&amp; Add to Album</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>AudioRepoDownloader</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
